ansible/roles/csm.ncn-initrd/tasks/main.yml

```diff
@@ -26,4 +26,4 @@
 
 - name: Create NCN initrd
   command: /srv/cray/scripts/common/create-ims-initrd.sh
-  when: cray_cfs_image|default(false)|bool
+
```

Comment:
You're broadening this task to run both during node personalization and image customization?

Reply:
The logic was: run this during image customization only.

Reply:
In case this wasn't clear Jason, we're moving away from using "when" but you specify hosts at the playbook level so there's nothing to replace the "when" with here.  This is just a role we'll only call during image customization.